### ISSUE TITLE
cannot customize xlabels/ylabels locations with SouthPolarStereo

### ISSUE THREAD

**Issue by jeffreymei** [SciTools/cartopy#1576]
### Description

So according to https://scitools.org.uk/cartopy/docs/v0.13/matplotlib/gridliner.html you can customize whether the right/left or top/bottom borders of the plot have tick labels on them by using `gl.xlabels_top = False`. But this gives no effect with SouthPolarStereo - it does work with Mercator.

#### Code to reproduce

```
ax = plt.subplot(111, projection=ccrs.SouthPolarStereo())
land = cfeature.NaturalEarthFeature('physical', 'land', '50m')
polar_crs = ccrs.SouthPolarStereo()
polar_extent = [-15, -120, -80, -66]
gl = ax.gridlines(color='lightgrey', linestyle='-', draw_labels=True, x_inline =False, y_inline =True)
ax.set_extent(polar_extent, crs=ccrs.PlateCarree())
gl.ylocator = mticker.FixedLocator([-85, -80, -75, -70, -65])
gl.xlocator = mticker.FixedLocator(np.arange(-160, 0, 20))
gl.ylabels_right = False
ax.xaxis.set_ticks([])
ax.add_feature(land, facecolor='#AAAAAA')
plt.show()
```
E.g. below, I don't want the longitude labels on the right border. 
![Figure_1](https://user-images.githubusercontent.com/3510683/83473258-251d2b00-a457-11ea-9097-63f833a26e72.png)

### Cartopy version
0.18, with Python 3.6.7.

**Comment by kdpenner** [comment]
This is a problem also with Albers Equal Area and the new `right_labels` and `top_labels` attributes.

```python
import numpy as np
import matplotlib.pyplot as plt
import matplotlib.ticker as mticker
import cartopy.crs as ccrs

plt_crs = ccrs.AlbersEqualArea(central_longitude=151.515,
                               central_latitude=53.27)
ax = plt.subplot(1, 1, 1, projection=plt_crs)
extent = [133.61, 169.42, 43.23, 63.31]
gl = ax.gridlines(draw_labels=True)
ax.set_extent(extent, crs=ccrs.PlateCarree())
gl.ylocator = mticker.FixedLocator(np.arange(43.23, 90, 10))
gl.xlocator = mticker.FixedLocator(np.arange(133.61, 180, 20))
gl.right_labels = False
gl.top_labels = False
plt.show()
```

![t](https://user-images.githubusercontent.com/9297904/84926458-6dfcf280-b099-11ea-8a0c-7b29edde5c7b.png)

**Comment by greglucas** [comment]
This is currently being worked on and discussed in the PR #1537. Also see #1530 for a similar request/discussion and feel free to weigh in with your thoughts on implementation details there as well.

A quick fix that I've used in the past is to add multiple gridliners to the axes and customize them with different lat/lon ranges and label visibilities, but it is a much more manual process of defining exactly the desired result, so not optimal.

**Comment by QuLogic** [comment]
Closed by #1537
